### PR TITLE
Add Windows stable support for app shortcuts

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This PR simply mentions Windows stable support for app shortcuts in https://web.dev/app-shortcuts

FIX: https://github.com/GoogleChrome/web.dev/issues/3228

Live preview at https://deploy-preview-3408--web-dev-staging.netlify.app/app-shortcuts/

FYI @jungkees @jgw96

### CODE REVIEW

**Comment by beaufortfrancois** [comment]
@kaycebasques Can we merge and push live July 22nd when Chrome 85 hits beta channel?

**Comment by kaycebasques** [comment]
@beaufortfrancois you should be able to merge this on July 22 (remember to remove the `DO NOT MERGE` label first). We don't have any automated mechanism for deploying the update on that date.

cc @robdodson interesting enough this is the second time today that the need for deploying a content update on a specific date has come up.

**Comment by jpmedley** [comment]
> cc @robdodson interesting enough this is the second time today that the need for deploying a content update on a specific date has come up.

This is a use case I have almost every time there's a new beta version of Chrome.

File: src/site/content/en/blog/app-shortcuts/index.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ authors:
   - beaufortfrancois
   - jungkees
 date: 2020-05-20
-updated: 2020-06-26
+updated: 2020-06-30
```
Comment (jpmedley) [inline src/site/content/en/blog/app-shortcuts/index.md:8]:
```suggestion
updated: 2020-07-22
```

**Comment by jungkees** [comment]
Looking great. Thanks for this update and reviewing, @beaufortfrancois, @jpmedley. Looking forward to seeing this merged on July 22.

**Comment by robdodson** [comment]
cc @robdodson interesting enough this is the second time today that the need for deploying a content update on a specific date has come up.

Does that mean deploying an update to a file that's already checked into the repo? If so, I'm not sure how best to enable that feature. Eleventy doesn't have partial builds—it builds the entire site whenever you do a build. So there isn't a good way to "hold a file back" from being deployed once it's been committed.

I think if we were using something with a database, like WordPress, it would be easier to enable this feature, but then the content of the site would no longer be available on GitHub.

**Comment by jpmedley** [comment]
>Does that mean deploying an update to a file that's already checked into the repo? If so, I'm not sure how best to enable that feature. Eleventy doesn't have partial builds—it builds the entire site whenever you do a build. So there isn't a good way to "hold a file back" from being deployed once it's been committed.

This should not have been merged until tomorrow. If I had seen that François had removed DO NOT MERGE I would have added it back.

**Comment by beaufortfrancois** [comment]
@jungkees Change is now live at https://web.dev/app-shortcuts/

**Pull request merged**